java/performance_test/src/main/resources/ray.conf

```diff
@@ -0,0 +1,5 @@
+ray {
+  job {
+    num-java-workers-per-process: 10
```

Comment:
We set `ray.job.num-java-workers-per-process` before `Ray.init()`. This file is unnecessary.